activemodel/lib/active_model/locale/en.yml

```diff
@@ -18,6 +18,7 @@ en:
       too_long:
         one: "is too long (maximum is 1 character)"
         other: "is too long (maximum is %{count} characters)"
+      too_long_in_bytes: "is too long (maximum is %{count} bytes)"
```

Comment:
`"is too long (maximum is %{count} bytes)"` seems too technical for an end-user readable error message, perhaps just `is too long`?

Reply:
but the whole point of this check is byte length, string itself can be short, but because of `fat` characters it cannot be proceeded.